tests/application/namespace-label/00-application.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Application
+metadata:
+  name: test-source
```

Comment:
nice if the application has a descriptive name like `namespace-label`

Reply:
Building a new iteration with more descriptive app-names